pkg/executor/distribute.go

```diff
@@ -55,6 +57,9 @@ func (e *DistributeTableExec) Open(context.Context) error {
 		return err
 	}
 	e.keyRanges = ranges
+	slices.SortFunc(e.partitionNames, func(i, j ast.CIStr) int {
```

Comment:
Why also sort `e.partitionNames`, does it has any relationship with `e.keyRanges` ?

Reply:
No， it relates to the alias. PD uses the alias as the unique job, but sometimes users the same inputs, such as partition(p1,p2) and partition(p2,p1). The key range of these two jobs is the same, but the aliases are different.